src/apps/answers/tests/test_answers.py

```diff
@@ -3517,7 +3517,7 @@ async def test_validate_multiinformant_assessment_fail_no_applet(self, client, l
 
         response = await client.get(url)
 
-        assert response.status_code == http.HTTPStatus.NOT_FOUND
+        assert response.status_code == http.HTTPStatus.BAD_REQUEST
```

Comment:
Again, I think `NOT_FOUND` is probably a more appropriate HTTP status for this case